tests/pkg_ucglib/main.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2018 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the Ucglib package.
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * @}
+ */
+
+#define TEST_OUTPUT_DUMMY 1
+#define TEST_OUTPUT_SDL 2
+#define TEST_OUTPUT_SPI 3
+
+#ifndef TEST_OUTPUT
+#error "TEST_OUTPUT not defined"
+#endif
+
+#if TEST_OUTPUT == TEST_OUTPUT_SPI
+#ifndef TEST_SPI
+#error "TEST_SPI not defined"
+#endif
+#ifndef TEST_DISPLAY
+#error "TEST_DISPLAY not defined"
+#endif
+#ifndef TEST_DISPLAY_EXT
+#error "TEST_DISPLAY_EXT not defined"
+#endif
+#ifndef TEST_PIN_CS
+#error "TEST_PIN_CS not defined"
+#endif
+#ifndef TEST_PIN_CD
+#error "TEST_PIN_CD not defined"
+#endif
+#ifndef TEST_PIN_RESET
+#error "TEST_PIN_RESET not defined"
+#endif
+#endif
+
+#include <stdio.h>
+
+#include "periph/gpio.h"
```

Comment:
If test_output is test_output_spi, then you'll also have to include "periph/spi.h"

Reply:
Fixed.